ui/yarn.lock

```diff
@@ -5157,11 +5157,16 @@ flush-write-stream@^1.0.0:
     inherits "^2.0.3"
     readable-stream "^2.3.6"
 
-follow-redirects@^1.0.0, follow-redirects@^1.10.0:
+follow-redirects@^1.0.0:
   version "1.14.8"
   resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.14.8.tgz#016996fb9a11a100566398b1c6839337d7bfa8fc"
   integrity sha512-1x0S9UVJHsQprFcEC/qnNzBLcIxsjAV905f/UkQxbclCsoTWlacCNOpQa/anodLl2uaEKFhfWOvM2Qg77+15zA==
 
+follow-redirects@^1.14.0:
+  version "1.15.0"
+  resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.15.0.tgz#06441868281c86d0dda4ad8bdaead2d02dca89d4"
+  integrity sha512-aExlJShTV4qOUOL7yF1U5tvLCB0xQuudbf6toyYA0E/acBNw71mvjFTnLaRp50aQaYocMR0a/RMMBIHeZnGyjQ==
```

Comment:
It differs from the one from dependabot:
https://github.com/dsaidgovsg/registrywatcher/pull/56/files#diff-3a968206d6de2fecfc5dacd7d94bab7744c9f5d5c999a816164d95cbc135c316R5160

Is this intentional?

Reply:
This results after running `yarn install` after making the dependabot changes. Might be dependency changes that are required?

Reply:
Ran `yarn install--frozen-lockfile` as suggested, the additional lines were not generated, hence the original changes from dependabot would also work

Reply:
Thanks for trying, `yarn` and `npm` seems notorious for changing lock file as default behaviour without flags haha.